pio/CMakeLists.txt

```diff
@@ -175,12 +175,8 @@ if("${CMAKE_Fortran_COMPILER_ID}" STREQUAL "GNU")
 endif()
 
 ADD_LIBRARY(pio ${SRCS_F90} ${SRCS_C} ${TEMPSRCF90})
-if(PNETCDF_LIBRARY)
-  TARGET_LINK_LIBRARIES(pio ${PNETCDF_LIBRARIES})
-ENDIF()
-if(NETCDF_LIBRARIES)
-  TARGET_LINK_LIBRARIES(pio ${NETCDF_LIBRARIES})
-ENDIF()
+TARGET_LINK_LIBRARIES(pio ${PnetCDF_Fortran_LIBRARIES})
```

Comment:
Can you add the conditionals (if(PnetCDF_Fortran_FOUND) and if(NetCDF_Fortran_FOUND) should work I think) back? PIO1 can be built without netcdf or pnetcdf libs (although i don't try it a binary format is supported by pio1).